lib/active_record/session_store.rb

```diff
@@ -33,6 +37,60 @@ def create_table!
         end
         connection.add_index table_name, session_id_column, :unique => true
       end
+
+      def determine_serializer
+        self.serializer ||= :marshal
```

Comment:
So the side effect of this line means that we're setting mattr upon the first call ... do you think that's okay?

Alternatively you can do this on line 44:

```ruby
when :marshal, nil
  #...